activity-handler.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
+
+var os = require("os"),
+    util = require("util");
+
+var async = require("async"),
+    debug = require("debug")("swfr:activity"),
+    humanize = require("humanize-plus");
+
+var activity = require("./activity"),
+    activities = require("./lib/activities");
+
+var activityHandler = function activityHandler(activitiesFolder, id) {
+  id = id | 1;
```

Comment:
Ah.  In that case, you mean `id = id || 1;`  `id | 1` is a variation on the JS idiom `str | 0` to coerce to an int.